.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn typescript:check
```

Comment:
to be honest, I don't think it would be needed. Let's remember there will be some course participants who will not be able to deal with all typescript errors :)

Reply:
changed to: yarn lint